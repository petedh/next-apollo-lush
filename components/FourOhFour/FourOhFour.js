const FourOhFour = () => {
  return (
    <div data-testid="is-FourOhFour">
      <img
        alt="404 image"
        src="https://res.cloudinary.com/lush/image/upload/f_auto,q_auto/v1542208897/lush_com/site_assets/404-Moving.gif"
        width={400}
        height={200}
      />
      <h3>Not Found</h3>
      <p>Sorry, we can&apos;t find the page you are looking forâ€¦</p>
    </div>
  )
}
export default FourOhFour
