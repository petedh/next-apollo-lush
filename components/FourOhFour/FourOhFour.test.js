import FourOhFour from './FourOhFour'
import renderer from 'react-test-renderer'
import '@testing-library/jest-dom'

it('renders FourOhFour unchanged', () => {
  const tree = renderer.create(<FourOhFour />).toJSON()
  expect(tree).toMatchSnapshot()
})
