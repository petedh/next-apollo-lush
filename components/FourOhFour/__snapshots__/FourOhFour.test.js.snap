// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FourOhFour unchanged 1`] = `
<div
  data-testid="is-FourOhFour"
>
  <img
    alt="404 image"
    height={200}
    src="https://res.cloudinary.com/lush/image/upload/f_auto,q_auto/v1542208897/lush_com/site_assets/404-Moving.gif"
    width={400}
  />
  <h3>
    Not Found
  </h3>
  <p>
    Sorry, we can't find the page you are looking forâ€¦
  </p>
</div>
`;
